models/comment.js

```diff
@@ -44,6 +46,38 @@ CommentSchema.statics.findByAssetId = function(asset_id) {
   return Comment.find({asset_id});
 };
 
+/**
+ * Change the status of a comment.
+ * @param {String} id  identifier of the comment  (uuid)
+ * @param {String} status the new status of the comment
+*/
+CommentSchema.statics.changeStatus = function(id, status) {
+  return Comment.update({'id': id}, {$set: {'status': status}}, {upsert: false}).then(() => {
```

Comment:
So you could change this to `Comment.findOneAndUpdate` [Model#findOneAndUpdate](http://mongoosejs.com/docs/api.html#query_Query-findOneAndUpdate) and pass `{new: true}` into the options hash and get back the updated comment for free, so you don't have to make a second db query.

So it would be `Comment.findOneAndUpdate({id}, {$set: {status}}, {upsert: true, new: true}).then(comment => {`